catalog/core/catalog-core-validationparser/src/main/java/ddf/catalog/validation/impl/ValidationParser.java

```diff
@@ -110,6 +116,8 @@
 
     private final Map<String, Changeset> changesetsByFile = new ConcurrentHashMap<>();
 
+    private static final ObjectMapper OBJECT_MAPPER = JsonFactory.create();
```

Comment:
I did a quick search online and looked at the code and it appears that `ObjectMapper` is probably thread-safe (and the static convenience methods on `JsonFactory` certainly treat it as such), but this still makes me a little nervous; there is no definitive comment/annotation in the boon code the assert that it is.

I think I'd feel more comfortable if you created stack-isolated `ObjectMapper`s at the top of each method that uses one. That would still be more efficient than creating one per call and would definitely assuage my concerns about thread safety.